pkg/logpoller/models.go

```diff
@@ -19,6 +20,7 @@ type Block struct {
 	BlockNumber          int64
 	BlockTimestamp       time.Time
 	FinalizedBlockNumber int64
+	SafeBlockNumber      sql.NullInt64
```

Comment:
My preference would be to keep this as int64, and fill values with 0 during db migration.
But not that strong opinion, so do what's more convenient.